### PR TITLE
[Main Course] - Abtest

### PR DESCRIPTION
**Pull request by jz263**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/Main_Course/Data_Analytics/2021-06-28-AB-Testing.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+https://www.udacity.com/course/ab-testing--ud257
+
+![A/B Testing 课程内容](https://cdn.jsdelivr.net/gh/BulletTech2021/Pics/2021-6-26/1624681931031-image.png)
+
+这门课会带你详细地讲解项目背景，真实案例中的实验设计，同时开放地鼓励你对实验结果进行阐述和分析。想积累经验的同学，建议亲自完成该课程，会发现很多实践中才会遇到的问题。想观望一下或者快速过一遍A/B测试流程的同学，可以参考这篇总结的测试报告。
+
+## 1. 实验背景和简介：
+
+优达学城是一个在线教育网站，该网站通过学生报名付费课程的方式盈利。优达学城将报名学生得课程完成率作为一个非常重要的运营指标，以此提高用户的使用满意度，获得较好的课程反馈和口碑。为了提高这一指标，优达学城计划对课程报名入口做一些改动，引导学生选择最合适的上课方式，提高课程的完成率。
```
Comment (jhqian0909) [inline docs/Main_Course/Data_Analytics/2021-06-28-AB-Testing.md:20]: 得课程-的课程

File: docs/Main_Course/Data_Analytics/2021-06-28-AB-Testing.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+现有的用户报名流程：
+
+在进行实验前，优达学城的课程主页上有两个选项：“开始免费试用” 和
+“获取课程资料”。如果学生点击“开始免费试用”，系统将要求输入银行卡信息，然后进入免费试学阶段，该阶段学生可以选择取消使用，否则14天后他们将自动收费。如果学生点击“获取课程资料”，那么他们可以免费观看视频以及做随堂测试，但是他们无法获得导师指导以及证书，也无法提交最终项目获得反馈。
+
+**1.2 对照组：**
+
+实验测试的报名流程：在实验中，如果学生单击“开始免费试用”，则会询问他们有多少时间可用于该课程。如果学生回复每周学习5个小时或以上，则将照常进入收费渠道录入流程。如果他们表示每周少于5个小时，则会出现一条消息，提示Udacity课程通常需要花费更多时间才能成功完成课程，并建议学生可能希望免费获取课程资料。此时，学生可以选择继续注册免费试用版，或改为免费获得课程材料。界面如下所示：
+
```
Comment (jhqian0909) [inline docs/Main_Course/Data_Analytics/2021-06-28-AB-Testing.md:32]: 图片呀

File: docs/Main_Course/Data_Analytics/2021-06-28-AB-Testing.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+**1.3 实验假设：**
+
+本次实验的假设是这会提前为学生设定更明确的期望，从而减少因为没有足够时间学习而退出免费试用，并因此受挫的学生数量，同时完成付费并完成课程的学生数量不会显著下降。如果假设成立，Udacity可以改善学生用户的整体体验，并提高导师指导可能完成课程学生的效率。
```
Comment (jhqian0909) [inline docs/Main_Course/Data_Analytics/2021-06-28-AB-Testing.md:35]: 提高导师指导可能完成课程学生的效率。这句有点没看明白

File: docs/Main_Course/Data_Analytics/2021-06-28-AB-Testing.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+| 流失率          | 0.096937   |
+| 点击-付费转化率 | 0.109313   |
+
+设定显著性水平 α = 0.05，检验功效 β = 0.20。
+
+根据样本计算公式：
+
+其中：
+
+页面浏览cookies所需的最小样本数量 =
```
Comment (jhqian0909) [inline docs/Main_Course/Data_Analytics/2021-06-28-AB-Testing.md:94]: 这里有点没看懂

File: docs/Main_Course/Data_Analytics/2021-06-28-AB-Testing.md (hunk 1 above)
Comment (jz263) [reply docs/Main_Course/Data_Analytics/2021-06-28-AB-Testing.md]: 已修改

File: docs/Main_Course/Data_Analytics/2021-06-28-AB-Testing.md (hunk 2 above)
Comment (jz263) [reply docs/Main_Course/Data_Analytics/2021-06-28-AB-Testing.md:32]: 已补充

File: docs/Main_Course/Data_Analytics/2021-06-28-AB-Testing.md (hunk 4 above)
Comment (jz263) [reply docs/Main_Course/Data_Analytics/2021-06-28-AB-Testing.md]: 这里有几个公式之前没转化成latex，现在补充上去了

**Review by Realvincentyuan — APPROVED** [review]

**Pull request merged**